File: POO/TP4/exercice2/Point.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package exercice2;

/**
 *
 * @author Lenovo
 */
public class Point {
    private double x;
    private double y;

    public Point(double x, double y) {
        this.x = x;
        this.y = y;
    }
    public void afficher(){
        System.out.println("Point( "+this.x+" , "+this.y+" )");
    }

    public double getX() {
        return x;
    }

    public double getY() {
        return y;
    }
    
    
}

File: 2021_2022/POO_CPP/TP2/EX7/Point.cpp
#include<iostream>
#include "Point.h"

Point::~Point()
{
    //dtor
}
Point::Point(float x, float y):x(x),y(y){
}

Point::Point(Point& p):x(p.x),y(p.y){
}

void Point::afficher() const
{
    std::cout << "P(" << x << "," << y << ")" << std::endl;
}
